packages/sanity/src/core/versions/components/GlobalBundleMenu.tsx

```diff
@@ -0,0 +1,136 @@
+import {AddIcon, CheckmarkIcon} from '@sanity/icons'
+import {Box, Button, Flex, Menu, MenuButton, MenuDivider, MenuItem, Text} from '@sanity/ui'
+import {useCallback, useContext, useState} from 'react'
+import {useRouter} from 'sanity/router'
+
+import {
+  VersionContext,
+  type VersionContextValue,
+} from '../../../_singletons/core/form/VersionContext'
+import {type Bundle, type Version} from '../types'
+import {BUNDLES, LATEST} from '../util/const'
+import {isDraftOrPublished} from '../util/dummyGetters'
+import {CreateBundleDialog} from './dialog/CreateBundleDialog'
+import {VersionBadge} from './VersionBadge'
+
+export function GlobalBundleMenu(): JSX.Element {
+  const router = useRouter()
+
+  // eslint-disable-next-line no-warning-comments
+  // FIXME REPLACE WHEN WE HAVE REAL DATA
+  const bundles = BUNDLES
+
+  // eslint-disable-next-line no-warning-comments
+  // FIXME REPLACE WHEN WE HAVE REAL DATA
+  const {currentVersion, setCurrentVersion, isDraft} =
+    useContext<VersionContextValue>(VersionContext)
+  const [createBundleDialogOpen, setCreateBundleDialogOpen] = useState(false)
+
+  const handleBundleChange = useCallback(
+    (bundle: Version) => () => {
+      const {name} = bundle
+
+      if (isDraftOrPublished(name)) {
+        router.navigateStickyParam('perspective', '')
```

Comment:
Does passing an empty string here mean it's no longer possible to access the system perspectives?

Reply:
Passing the empty string here is how we were doing it when it's either the Draft or Published :)

This is just the fancy way of doing what we were doing in the StudioNavbar.tsx before my component.
So it just removes the latest part of the url

<img width="503" alt="image" src="https://github.com/sanity-io/sanity/assets/6951139/aed4dd6c-6ab9-40fa-913a-900a101a8be5">

Reply:
Ah, my bad! 🙂